### ISSUE TITLE
Keep plugin UI accessibile

### ISSUE THREAD

**Issue by mlewand** [cksource/ckeditor-plugin-a11ychecker#22]
# Overview

We can't forget to keep the plugin UI accessible.

This will especially come down to handling QuickFix form and issues navigation.
## Issues Navigation

Using a screen reader user should be able to determine:
- What kind of an issue is focused.
- What element caused the issue?
  - I think we should inform about element type: header, paragraph, list item.
  - We should read issued element inner text.
- How many issues remains.
- Are there any QuickFixes available.
## QuickFix Form

We need to put extra attention to error reporting.

In case of invalid value in any field, user should be notified about the error. Possibly we should also move focus to the first problematic field.
